schemas/2018-09-07-preview/Microsoft.Kusto.json

```diff
@@ -0,0 +1,174 @@
+﻿{
+  "id": "https://schema.management.azure.com/schemas/2018-09-07-preview/Microsoft.Kusto.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Kusto",
+  "description": "Microsoft Kusto Resource Types",
+  "resourceDefinitions": {
+    "clusters": {
```

Comment:
>clusters [](start = 5, length = 8)

please add tests for the new resource types.

Reply:
Done